docs/deployment-guide.md

```diff
@@ -833,6 +834,18 @@ In that case, you can add the `cluster_keepoff` variable to one of the `reposito
 
 > A typical use case is to have a dedicated Solr tracking node. The playbook will then prefer to use that dedicated node - if it finds one - for solr tracking and only use the other as backup server (no load balancing)
 
+## Maintenance
+
+After the initial deploy, may arise the need to execute maintenance tasks that are handled via specific playbooks.
+
+### Search Enterprise Reindexing
+
+You can trigger the reindexing of existing content in Search Enterprise using a dedicated playbook:
+
+```bash
+pipenv run ansible-playbook playbooks/search-enterprise-reindex.yml -i <inventory_file>.yml
```

Comment:
We should add a warning message that if the reindex is triggerred on an index that has already indexed content that may/will create duplicates...
Wondering if we should also add a safe guard in the service file so it's not fired up manually by an admin who may actually want to start the connector itself?

Reply:
I think that this type of concerns should be documented somewhere like https://docs.alfresco.com/search-enterprise/latest/admin/ and not here

can't find anything related to duplication search in the docs

Reply:
anyway I added support for specifying additional parameters for the reindex job so it's possibile to do partial reindexing as per docs

Reply:
I'm still concerned about someone mistakenly triggering the reindex through a `systemctl` command.
We could just check for existence of a file or var before starting that process (ofc the playbook would need to set things up when invoked on purpose).

Reply:
engineers reported that running multiple times the reindex is not a problem, so we should be good to go as it is